Backend/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Please delete this file. We are not open source. We are closed private company. 
Adding a license means we are ok that any one could use your app.